#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-noroms.dpatch by Stuart Brady <sdbrady@ntlworld.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't install the ROMs, as they are non-free.

--- fuse-emulator-0.9.0-old/roms/Makefile.am	2008-01-13 19:31:46.000000000 +0000
+++ fuse-emulator-0.9.0-new/roms/Makefile.am	2008-01-12 22:02:01.000000000 +0000
@@ -36,6 +36,6 @@
 	if1-1.rom if1-2.rom \
 	plusd.rom
 
-pkgdata_DATA = $(ROMS)
+#pkgdata_DATA = $(ROMS)
 
 EXTRA_DIST = $(ROMS)
